src/bcn/bc1.rs

```diff
@@ -37,7 +37,7 @@ pub fn decode_bc1_block(data: &[u8], outbuf: &mut [u32]) {
             ((b0 + b1) / 2) as u8,
             255,
         );
-        c[3] = color(0, 0, 0, 255);
+        c[3] = color(0, 0, 0, 0);
```

Comment:
The alpha value should be decided via an input value.
Usually BC1 is just RGB.
BC1a is "just" optional and might not be the intended output.